include/riak_kv_ts.hrl

```diff
@@ -71,8 +71,8 @@
           'SELECT'              :: #riak_sel_clause_v1{},
           'FROM'        = <<>>  :: binary() | {list, [binary()]} | {regex, list()},
           'WHERE'       = []    :: [riak_ql_ddl:filter()],
-          'ORDER BY'    = []    :: [riak_kv_qry_compiler:sorter()],
-          'LIMIT'       = []    :: [riak_kv_qry_compiler:limit()],
+          'ORDER BY'            :: undefined | [riak_kv_qry_compiler:sorter()],
+          'LIMIT'               :: undefined | riak_kv_qry_compiler:limit(),
```

Comment:
The composition of the record did not change, only the default values did. Because 1.4 nodes don't inspect `'ORDER BY'` and `'LIMIT'`, I don't expect things will break because of this change. An upgrade/downgrade test is in the plans.